### PR TITLE
Remove dev page from Survey Runner application

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
This removed the dev page (/dev) from the survey runner application. This is a move to making the test environments behave more like the production environment where the token is generated on a remote system and passed across by the user.

### How to review 
Follow the README to run survey runner and check that everything behaves as expected.
Check that /dev is no longer accessible.

### CODE REVIEW

File: app/settings.py
Comment (collisdigital) [inline app/settings.py]: use `with` instead of `open` and `close` ?

**Comment by dcdarrell9** [comment]
Can run and make any dev changes pretty easy, everything as expected, all seems to be good 👍

File: docker-compose.yml
Comment (ajmaddaford) [inline docker-compose.yml]: We should move this to the ONSDigital github before merging this.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: docker-compose.yml
Comment (jonnyshaw89) [reply docker-compose.yml]: @collisdigital can you transfer ownership?

**Review by adoublebarrel — APPROVED** [review]

File: README.md
Comment (ajmaddaford) [inline README.md]: Navigate -> navigate

**Comment by yoganandkunche** [comment]
LGTM :+1:

File: README.md
Comment (ajmaddaford) [inline README.md]: I think it would be useful to provide the docker command here to run the latest launcher

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
Looks good, we can merge this once we get https://github.com/ONSdigital/go-launch-a-survey running with a valid certificate and redirecting to the correct URL in pre-prod

**Review by dcdarrell9 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**
